tensorflow/workspace2.bzl

```diff
@@ -325,10 +325,10 @@ def _tf_repositories():
     tf_http_archive(
         name = "org_sqlite",
         build_file = "//third_party:sqlite.BUILD",
-        sha256 = "49112cc7328392aa4e3e5dae0b2f6736d0153430143d21f69327788ff4efe734",
-        strip_prefix = "sqlite-amalgamation-3400100",
+        sha256 = "1286856f5dff3b145f02dfe5aa626657c172a415889e5a265dc4dee83a799c03",
```

Comment:
Are you sure this is the right checksum?

```
Checksum was bb5849ae4d7129c09d20596379a0b3f7b1ac59cf9998eba5ef283ea9b6c000a5 but wanted 1286856f5dff3b145f02dfe5aa626657c172a415889e5a265dc4dee83a799c03
```

Reply:
You're right, updated with the correct checksum.

Reply:
Thank you, running a new import and trying to shepherd this in